sklearn/decomposition/tests/test_pca.py

```diff
@@ -248,21 +248,27 @@ def test_pca_inverse(svd_solver, whiten):
             "auto",
             -1,
             (
-                r"n_components={}L? must be between {}L? and "
-                r"min\(n_samples, n_features\)={}L? with "
-                r"svd_solver=\'{}\'"
+                r"The 'n_components' parameter of PCA must be an int "
+                r"in the range \[0, inf\), a float in the "
+                r"range \(0, 1\), a str among {'mle'} or None. Got -1 instead."
```

Comment:
you can just remove this tuple from the test since it's now covered by common tests